src/extras/gizmo/gizmo.js

```diff
@@ -227,22 +230,45 @@ class Gizmo extends EventHandler {
      */
     intersectData = [];
 
+    /**
+     * Creates a new gizmo layer and adds it to the scene.
+     *
+     * @param {AppBase} app - The app.
+     * @param {string} [layerName] - The layer name. Defaults to 'Gizmo'.
+     * @param {number} [layerIndex] - The layer index. Defaults to -1.
```

Comment:
clarify what happens in the default case (added to the end?)
maybe use undefined instead, does not need a default
then the code can be 
const index = layerIndex ?? app.scene.layers.layerList.length;

Reply:
The convention is that index -1 is the end of the list so I would think thats clear from the default but I can make it more explicit

Reply:
mention it in the JSDoc please.